bikeshed/markdown.py

```diff
@@ -67,6 +67,8 @@ def tokenizeLines(lines, numSpacesForIndentation, features=None, opaqueElements=
 			match = re.search(r"\{#([^ }]+)\}\s*$", line)
 			if match:
 				token['id'] = match.group(1)
+		elif re.match(r"([*_-])\s*\1\s*\1", line):
```

Comment:
Need a `(\s*\1)*$` at the end to ensure that the line doesn't have anything else on it, per CommonMark.